styles/styles.css

```diff
@@ -375,6 +375,11 @@ main {
 }
 
 main .section {
+  padding: 45px 30px 65px;
+  overflow: hidden;
```

Comment:
padding: 45px 30px 65px;?
I guess that is the general padding of the main sections?
Or which one are you talking about?

Reply:
my comment refers to the `overflow: hidden`

Reply:
I didn't put that one but I see it is used on the original site, probably for lower screen widths to avoid having scrollbars:
<img width="377" alt="image" src="https://user-images.githubusercontent.com/44608307/216656787-29d1a45c-5a27-4072-8fcd-6b3cc274ee00.png">